### DISCUSSION TITLE
PVE 7.4-3 and Debian 12 based LXC's

### DISCUSSION THREAD

**Discussion by EricSnijders** [tteck/Proxmox#1686] [Help Needed] [answered]
I'm a bit lost. So i've just read/heard about an issue regarding Debian 12 based LXC's on PVE 7.x. What is the best thing to do? Should i update PVE to version 8? Or is there anyway i can still use some "older" install script to get the specific LXC based on Debian 11? I'm not an expert on GitHub. I've found the script (in this case for NginX Proxy Manager), but i'm having some trouble finding the right script version based on Debian 11.

**Comment by tteck** [comment, marked as answer]
Choose the "Advanced" settings and opt for Debian 11.

**Reply by EricSnijders** [reply]
Wow, thanks for the fast reply. The warning during the LXC deployment was cut short, so i didn't know what to do. Thanks for everything you do!

**Comment by tteck** [comment]
Even more concerning is the usage of an outdated Proxmox version (PVE 7.4-3).
I believe that the latest version for PVE 7 is 7.4-16.
